tests/framework/helpers/ArrayHelperTest.php

```diff
@@ -1269,5 +1269,15 @@ public function testFilter()
         $this->assertEquals(ArrayHelper::filter($array, ['X']), []);
         $this->assertEquals(ArrayHelper::filter($array, ['X.Y']), []);
         $this->assertEquals(ArrayHelper::filter($array, ['A.X']), []);
+        
+        $tmp = [
+            'a' => 0,
+            'b' => '',
+            'c' => false,
+            'd' => null,
+            'e' => true
```

Comment:
@hhniao and also I can propose to put here comma after element too, it's about Yii2 code standard rules, in particular: https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md#associative (keep attention on commas).